flex-templates/java/regional_dlp_transform/pom.xml

```diff
@@ -0,0 +1,481 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2021 Google Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.23</version>
+  </parent>
+
+  <groupId>com.google.cloud.blueprints.datawarehouse</groupId>
+  <artifactId>regional-bq-dlp-bq-streaming</artifactId>
+  <version>1.0.0</version>
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <beam.version>2.34.0</beam.version>
+
+    <maven-enforcer-plugin.version>3.0.0</maven-enforcer-plugin.version>
+    <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+    <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
+    <maven-exec-plugin.version>3.0.0</maven-exec-plugin.version>
+
+
+    <slf4j.version>1.7.32</slf4j.version>
+    <google.guava.version>29.0-jre</google.guava.version>
+
+    <apache-http-client-v2>1.39.2</apache-http-client-v2>
+    <avro.version>1.8.2</avro.version>
+    <beam.version>2.34.0</beam.version>
+    <beam-vendor-guava.version>0.1</beam-vendor-guava.version>
+    <extra.enforcer.rules.version>1.3</extra.enforcer.rules.version>
+    <bigtable.version>1.15.0</bigtable.version>
```

Comment:
similar comments from the other pom.xml file.  mabye copy the edits from that one to here, as well?